### ISSUE TITLE
Download blocked by Hamburg Regional Court

### ISSUE THREAD

**Issue by Morasithil** [safing/portmaster#1366]
**Pre-Submit Checklist**:

- Check applicable sources for existing issues:
  - [Windows Known Issues](https://docs.safing.io/portmaster/install/windows#known-issues)
  - [Linux Known Issues](https://docs.safing.io/portmaster/install/linux#compatibility)
  - [Github Issues](https://github.com/safing/portmaster/issues?q=is%3Aissue+label%3Abug)

**What happened**:
This might not be a bug, but could be eventually.

**What did you expect to happen?**:
I'm trying to download the youtube downloader known as yt-dl from github, https://ytdl-org.github.io/youtube-dl/download.html
The site redirects me to this link https://ytdl-org.github.io/youtube-dl/download.html
And then i end up here, https://yt-dl.org/downloads/2021.12.17/youtube-dl.exe
However i cannot download the software, this is what happens:
Access denied
Due to a [ruling] (https://openjur.de/u/2466945.html) of the Hamburg Regional Court, access to this website is blocked.
Zugriff gesperrt
Aufgrund eines [Urteils](https://openjur.de/u/2466945.html) des Landgerichts Hamburg ist der Zugriff auf diese Website gesperrt.

I don't get it, i have tried all dns servers that are available in portmaster and tried two vpns but the site is always blocked, how can that even be?! The court of hambourg germany doesn't have power over swiss vpn servers or the cloudflare dns provider, does it?!
This censorship is shocking.

**How did you reproduce it?**:

**Debug Information**:

**Comment by Raphty** [comment]
this can have a lot of reason, it might be caches in the browser, or as you are suggesting DNS related or it can be the VPN.... 

Maybe the website is doing something themselves? and look for regional data like the language in your browser....

**Comment by Morasithil** [comment]
> this can have a lot of reason, it might be caches in the browser, or as you are suggesting DNS related or it can be the VPN....
> 
> Maybe the website is doing something themselves? and look for regional data like the language in your browser....

I figured out that the website indeed has been blocked due to a ruling of the german court.
What i don't understand why i can't bypass this blockage by using another dns provider or vpn provider.
Unless the site has literally been taken down by the court and that's why i can only see this page as a result.
It's like when an illegal darknet market get's taken down the site remains online but it says something like "this domain has been seized by the fbi, europol, etc".

I found a working fork to the youtube downloader, a brilliant piece of software!
https://github.com/yt-dlp/yt-dlp/releases
